### PR TITLE
CCM3-19 - add GEFF tool

### PR DESCRIPTION
**Pull request by gsilver**

### CODE REVIEW

File: src/main/webapp/assets-lti/js/app/app.js [hunk 1]
```diff
@@ -23,6 +23,10 @@ canvasSupportApp.config(['$routeProvider',
       templateUrl: 'assets-lti/views/course.html',
       controller: 'courseController'
     }).
+    when('/grades', {
+      templateUrl: 'assets-lti/views/grades.html',
+      controller: 'gradesController'
```
Comment (pushyamig) [inline src/main/webapp/assets-lti/js/app/app.js]: Where is grades.html? Is that checkin already

File: src/main/webapp/index-lti.vm [hunk 2]
```diff
@@ -49,15 +49,18 @@
...
+                    <a ng-if="view && view !=='LTI'" class="collapse-nav"  ng-click="changeView('LTI')" href="#ltihome">LTI</a>
+                  </li>
+                  <li ng-if="ltiLaunch.role_is_account_admin ==='true'">
+                    <a ng-if="view !=='Sub Account Administrator Utils'" class="collapse-nav" ng-click="changeView('Sub Account Administrator Utils')"  href="#saa">SAA</a>
+                  </li>
+                  <li ng-if="view !=='Grades'">
+                    <a  class="collapse-nav" ng-click="changeView('Grades')"  href="#grades">Grades</a>
+                  </li>
+                  <li>
+                    <a href="" class="collapse-nav" id="helpLink" data-url="help-lti.html">Help</a>
```
Comment (pushyamig) [inline src/main/webapp/index-lti.vm]: is GEFF tool is available to instructors Who are registered as Mpathways?

Comment (gsilver) [inline src/main/webapp/index-lti.vm]: Oooh. Great question.

File: src/main/webapp/assets-lti/js/app/app.js (hunk 1 above)
Comment (gsilver) [inline src/main/webapp/assets-lti/js/app/app.js]: Oops!

Comment (gsilver) [inline src/main/webapp/assets-lti/js/app/app.js]: Oops.

**Pull request merged**